resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/LoggingPodStatusWatcher.scala

```diff
@@ -77,9 +83,19 @@ private[k8s] class LoggingPodStatusWatcherImpl(
     }
   }
 
+  override def reset(): Unit = {
+    resourceTooOldReceived = false
+  }
+
   override def onClose(e: KubernetesClientException): Unit = {
-    logDebug(s"Stopping watching application $appId with last-observed phase $phase")
-    closeWatch()
+    logInfo(s"Stopping watching application $appId with last-observed phase $phase")
```

Comment:
This is a regression because this is previous `logDebug`. (Of course, this is different from `master` branch, too).

Reply:
Done